package.json

```diff
@@ -54,6 +54,7 @@
         "prettier": "^2.5.1",
         "pretty-quick": "^3.1.3",
         "size-limit": "^8.1.0",
+        "supertokens-node": "latest",
```

Comment:
won't work in cicd for now case this sdk hasn't been released yet.

Reply:
i also don’t like to have the node sdk in this here cause we have to rememeber to change the versions etc.. it’s enough to just hard code the expected values in the tests i think, as opposed to take from node sdk